src/OpenTelemetry/Trace/ExceptionProcessor.cs

```diff
@@ -19,6 +19,22 @@ internal sealed class ExceptionProcessor : BaseProcessor<Activity>
     public ExceptionProcessor()
     {
 #if NET6_0_OR_GREATER || NETFRAMEWORK
+        bool getExceptionPointersSupported = true;
+
+        try
+        {
+            Marshal.GetExceptionPointers();
+        }
+        catch
```

Comment:
Any specific exception type here?

Reply:
Yeah... need to check whether PlatformNotSupportedException is the only type that would be thrown here.